### PR TITLE
Various wording improvements

### PR DESCRIPTION
**Pull request by mnapoli**
Steven tried the whole "getting started" experience and noted some feedback on different wording (docs, CLI text, etc.).

This PR contains various wording improvements.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -104,7 +104,9 @@ If the templates proposed by the `serverless` command do not fit your needs, you
 
 ```sh
 # replace folder-name below with the folder name of the example you want to use
-$ serverless create -u https://github.com/serverless/examples/tree/master/folder-name -n my-project
+$ serverless create \
+  -u https://github.com/serverless/examples/tree/master/folder-name \
+  -n my-project
```
Comment (mnapoli) [inline docs/getting-started.md:109]: Avoid horizontal scrollbar, as visible here: https://www.serverless.com/framework/docs/getting-started#getting-started

File: scripts/serverless.js [hunk 2]
```diff
@@ -173,10 +173,10 @@ const processSpanPromise = (async () => {
...
         `"${humanizedPropertyPath}" is not accessible ` +
           '(configured behind variables which cannot be resolved at this stage).\n' +
-          'Starting with next major release, ' +
-          'this will be communicated with a thrown error.\n' +
-          'Set "variablesResolutionMode: 20210326" in your service config, ' +
-          'to adapt to this behavior now',
+          'In the next major version ' +
+          'this will result in an error.\n' +
+          'Set "variablesResolutionMode: 20210326" in your service config ' +
+          'to opt in this behavior now',
```
Comment (pgrzesik) [inline scripts/serverless.js:179]: I think it should say "opt in to this behavior" instead of current version - could you confirm that?

**Review by pgrzesik — COMMENTED** [review]
Looks good in general, I have only one question about wording

File: scripts/serverless.js (hunk 2 above)
Comment (mnapoli) [reply scripts/serverless.js]: 👍  fixed, thanks!

**Review by medikoo — APPROVED** [review]
Great improvement 👍

**Review by pgrzesik — APPROVED** [review]
Looks great 👍

**Pull request merged**